faunadb-java-dsl/src/main/java/com/faunadb/client/errors/FaunaException.java

```diff
@@ -21,6 +21,11 @@ public FaunaException(String message) {
     this.response = Optional.absent();
   }
 
+  public FaunaException(String message, Throwable ex) {
```

Comment:
Maybe rename `ex` to `cause`? That's usually the term used to refer to other errors.